src/nexus/GUI.py

```diff
@@ -84,12 +101,12 @@ def stop_logging(self):
 
     def start_stop(self):
         """Controller for start/stop logging button"""
-        if self.start_stop_button.text() == "Start logging":
+        if self.start_stop_button.text() == self.translator.translate("GUI", "Start logging"):
```

Comment:
Can we use `self.tr` instead of `self.translator.translate` for all of these?

Reply:
We do not have `self.tr` here, `self.translator` needs to be used.

Reply:
Couldn't we bind the function? 🙂